browser.js

```diff
@@ -0,0 +1,3 @@
+(() => {
```

Comment:
What do you think about not add this file to versioning, it's not better to build it with CI and upload to our package manger when we create a new version?

Reply:
sorry bro, didn't want to add this file